cfme/fixtures/ansible_fixtures.py

```diff
@@ -101,6 +102,26 @@ def ansible_service_catalog(appliance, ansible_catalog_item, ansible_catalog):
     return service_catalog_
 
 
+@pytest.fixture
+def ansible_service_request(appliance, ansible_catalog_item):
+    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
+    service_request_ = appliance.collections.requests.instantiate(description=request_descr)
+    yield service_request_
+
+    if service_request_.exists():
+        service_id = appliance.rest_api.collections.service_requests.get(description=request_descr)
+        appliance.rest_api.collections.service_requests.action.delete(id=service_id.id)
+
+
+@pytest.fixture
+def ansible_service(appliance, ansible_catalog_item):
+    service_ = MyService(appliance, ansible_catalog_item.name)
```

Comment:
Is there any issue with naming the variable "service" ? 
If not please use :
service = MyService(appliance, ansible_catalog_item.name)

Reply:
I will update it.